docs/VictoriaLogs/README.md

```diff
@@ -23,263 +23,18 @@ VictoriaLogs is at Preview stage now. It is recommended evaluating it on product
-
-The `_msg_field` parameter must contain the field name with the log message generated by Logstash. This is usually `message` field.
-See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#message-field) for details.
-
-The `_time_field` parameter must contain the field name with the log timestamp generated by Logstash. This is usually `@timestamp` field.
-See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#time-field) for details.
-
-It is recommended specifying comma-separated list of field names, which uniquely identify every log stream collected by Logstash, in the `_stream_fields` parameter.
-See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#stream-fields) for details.
-
-If some [log fields](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#data-model) aren't needed,
-then VictoriaLogs can be instructed to ignore them during data ingestion - just pass `ignore_fields` parameter with comma-separated list of fields to ignore.
-For example, the following config instructs VictoriaLogs to ignore `log.offset` and `event.original` fields in the ingested logs:
-
-```conf
-output {
-  elasticsearch {
-    hosts => ["http://localhost:9428/insert/elasticsearch/"]
-    parameters => {
-        "_msg_field" => "message"
-        "_time_field" => "@timestamp"
-        "_stream_fields" => "host.hostname,process.name"
-        "ignore_fields" => "log.offset,event.original"
-    }
-  }
-}
-```
-
-If the Logstash sends logs to VictoriaLogs in another datacenter, then it may be useful enabling data compression via `http_compression: true` option.
-This usually allows saving network bandwidth and costs by up to 5 times:
-
-```conf
-output {
-  elasticsearch {
-    hosts => ["http://localhost:9428/insert/elasticsearch/"]
-    parameters => {
-        "_msg_field" => "message"
-        "_time_field" => "@timestamp"
-        "_stream_fields" => "host.hostname,process.name"
-    }
-    http_compression => true
-  }
-}
-```
-
-By default the ingested logs are stored in the `(AccountID=0, ProjectID=0)` [tenant](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#multitenancy).
-If you need storing logs in other tenant, then specify the needed tenant via `custom_headers` at `output.elasticsearch` section.
-For example, the following `logstash.conf` config instructs Logstash to store the data to `(AccountID=12, ProjectID=34)` tenant:
-
-```conf
-output {
-  elasticsearch {
-    hosts => ["http://localhost:9428/insert/elasticsearch/"]
-    custom_headers => {
-        "AccountID" => "1"
-        "ProjectID" => "2"
-    }
-    parameters => {
-        "_msg_field" => "message"
-        "_time_field" => "@timestamp"
-        "_stream_fields" => "host.hostname,process.name"
-    }
-  }
-}
-```
+VictoriaLogs can accept logs from popular log collectors via following data ingestion protocols:
```

Comment:
Please list the protocols here and provide links to collectors.md to each type of collector